lib/kramdown/parser/gfm/emoji_parser.rb

```diff
@@ -15,7 +15,7 @@ class GFM
 
       EMOJI_NAMES   = Emoji.all.flat_map(&:aliases).freeze
       REGISTRY      = EMOJI_NAMES.zip(EMOJI_NAMES).to_h.freeze
-      EMOJI_PATTERN = /:(\w+):/
+      EMOJI_PATTERN = /:([\w\+]+):/
```

Comment:
This is not covering 👎🏾 / `-` is not part of the character class matched by `\w`, and you shouldn't need to escape `+` if it's _inside_ of a character class (`[ ] \ - ^` are the characters that would need escaping):
```suggestion
      EMOJI_PATTERN = /:([\w\-+]+):/
```

That said, there are a lot of places that don't support +1/-1, they are effectively aliases that were added to [gemoji] that map to the actual unicode CLDR short names (`thumbs up` and `thumbs down`)
[gemoji]: https://github.com/github/gemoji

Reply:
It might be beneficial to simply rely on an emoji regex gem to supply the regex.
Example: https://github.com/janlelis/unicode-emoji